JS/audio.js

```diff
@@ -0,0 +1,37 @@
+function listenKeypress() {
+    document.addEventListener('keypress', function(e) {
+        var keys = {
+            'q': '1',
+            'w': '2',
+            'e': '3',
+            'r': '4'
+        }
+        if (e.key in keys) {
+            var sound = 'cheer' + keys[e.key]
+            makeNoise(sound)
+            var $city = $('#city-' + keys[e.key])
+            shootLaser($city)
+        }
+    })
+}
+
+var cities = document.querySelectorAll('.city')
+cities.forEach(function(city) {
+    city.addEventListener('click', function() {
+        var param = 'cheer' + city.dataset.city
```

Comment:
if you make the keys in the array 1,2,3,4 you wont need to add cheer to the string.

Reply:
Eventually we will have more sounds in the makeNoise() function so it helps to keep the prefix. We also might (and probably will) replace the cheering with a laser noise since that would be more fitting

Reply:
see
https://github.com/Mayden-Academy/2018-rocket-man/blob/story-4-miss-note/JS/audio.js